.github/workflows/auto_update.yml

```diff
@@ -0,0 +1,49 @@
+name: Update DB
+
+on: workflow_dispatch
+
+jobs:
+  update_db:
+    name: "Update DB"
+    runs-on: ubuntu-latest
+    env:
+      IMAGEBUILDER_BOT_GITLAB_SSH_KEY: ${{ secrets.IMAGEBUILDER_BOT_GITLAB_SSH_KEY }}
+      TRIGGER_TOKEN: ${{ secrets.TRIGGER_TOKEN }}
+    steps:
+      - name: Clone composer
+        uses: actions/checkout@v3
+        with:
+          repository: osbuild/osbuild-composer
+          path: composer
+
+      - name: Checkout manifest-db
+        uses: actions/checkout@v3
+        with:
+          path: manifest-db
+          fetch-depth: 0
+
+      - name: Update DB
+        run: |
+          ./manifest-db/tools/import-image-tests composer/test/data/manifests/ manifest-db/manifest-db --manifest-only --db-ignore=manifest-db/db-ignore --verbose
+
+      - name: Send to CI
+        run: |
+          cd manifest-db
+          git config --local user.name "SchutzBot"
+          git config --local user.email "imagebuilder-bots+schutzbot@redhat.com"
+
+          now=$(date '+%Y-%m-%d-%H%M%S')
+          BRANCH_NAME="db-update-$now"
+          git checkout -b "$BRANCH_NAME"
+          git add -A
+          git commit -m "db: automatic update of manifests."
+
+          mkdir -p ~/.ssh
+          echo "${IMAGEBUILDER_BOT_GITLAB_SSH_KEY}" > ~/.ssh/id_rsa
+          chmod 400 ~/.ssh/id_rsa
+          touch ~/.ssh/known_hosts
+          ssh-keyscan -t rsa gitlab.com >> ~/.ssh/known_hosts
+          git remote add ci git@gitlab.com:redhat/services/products/image-builder/ci/manifest-db.git
+          git push -f ci
```

Comment:
Doesn't this create a "regression" gitlab pipeline? Maybe a `ci.skip` would be nice here?

https://docs.gitlab.com/ee/user/project/push_options.html#push-options-for-gitlab-cicd